homeassistant/components/shelly/__init__.py

```diff
@@ -122,7 +134,6 @@ async def shutdown(self):
         if self._unsub_stop:
```

Comment:
You can remove this whole shutdown in the device wrapper and the listening for HA stop too.

Reply:
Done